documentation/docs/14-configuration.md

```diff
@@ -163,6 +164,7 @@ An object containing zero or more of the following `string` values:
 
 See [Prerendering](#ssr-and-javascript-prerender). An object containing zero or more of the following:
 
+- `concurrency` — how many pages can be prerendered simultaneously. In cases where prerendering performance is network-bound (for example loading content from a remote CMS) this can speed things up
```

Comment:
In my Java mindset, I was thinking maybe this should be called `threads` or something more specific, but then I remembered the JS is single threaded and it's the event loop we're taking advantage of. Maybe it's worth adding a bit more detail to the docs?
```suggestion
- `concurrency` — how many pages can be prerendered simultaneously. JS is single-threaded, but in cases where prerendering performance is network-bound (for example loading content from a remote CMS) this can speed things up by processing other tasks while waiting on the network response
```

Reply:
Multi-threading is possible in JS with workers.

On that matter https://github.com/Aslemammad/tinypool looks like an interesting library (even if it is way more complex than the queue implemented in this PR). It is used by [vitest](https://vitest.dev/) to run tests in parallel.